docs/proposals/20220927-label-sync-between-machine-and-nodes.md

```diff
@@ -98,14 +99,18 @@ While designing a solution for syncing labels between Machine and underlying Kub
 
 ### Label domains & prefixes
 
-The idea of scoping synchronization to a well defined set of labels is a first answer to security/concurrency concerns; labels to be managed by Cluster API have been selected based on following criteria:
+The list of labels that will be synced between the Machines and the Nodes is configured by a flag on the manager that list the domains or subdomains a label should belong to.
 
-- The `node-role.kubernetes.io` label has been used widely in the past to identify the role of a Kubernetes Node (e.g. `node-role.kubernetes.io/worker=''`). For example, `kubectl get node` looks for this specific label when displaying the role to the user.
+If no list is provided it defaults to:
```

Comment:
Might be we should rephrase this sentence and the one above to better express the fact that the flags values are additive to defaults, and drop the note in L113